PurchasesTests/Purchasing/PurchasesTests.swift

```diff
@@ -252,9 +252,9 @@ class PurchasesTests: XCTestCase {
     }
     
     func testUsingSharedInstanceWithoutInitializingRaisesException() {
-        expect{ Purchases.shared }.to(raiseException())
+        expectToThrowException(.parameterAssert) { _ = Purchases.shared }
```

Comment:
out of curiosity, why the `_ =` part?

Reply:
`Purchases.shared` isn't marked as `@discardableResult` so we get a warning about not using the return value. I definitely think we shouldn't use `@discardableResult` for this property, so `_ =` silences the warning